### PR TITLE
feat(storybook): update storybook to v6 - FE-3333

### PR DESCRIPTION
**Pull request by mkrds**
### Proposed behaviour
This PR updates Storybook to the newest v6

### Checklist

- [x] Commits follow our style guide
- [x] Screenshots are included in the PR if useful
- [x] All themes are supported if required
- [x] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [x] Storybook added or updated if required
- [x] Typescript `d.ts` file added or updated if required
- [x] Carbon implementation and Design System documentation are congruent

### CODE REVIEW

File: .storybook/preview.js [hunk 1]
```diff
@@ -61,14 +56,18 @@ const customViewports = {
...
+  },
+  layout: "fullscreen",
   options: {
     isFullscreen: false,
     panelPosition: "bottom",
     showNav: true,
     showPanel: true,
     theme: sageTheme,
-    storySort: (a, b) => a[1].id.localeCompare(b[1].id),
+    // storySort: (a, b) => a[1].id.localeCompare(b[1].id),
```
Comment (sjohnson-sage) [inline .storybook/preview.js:70]: Can we just delete this line?

**Review by sjohnson-sage — COMMENTED** [review]
Looks good to me, one very minor comment

File: cypress/fixtures/commonComponents/confirm.json [hunk 2]
```diff
@@ -1,105 +1,80 @@
...
-    "open": true,
     "iconType": "error"
   },
   "iconWarning": {
-    "open": true,
     "iconType": "warning"
   },
   "iconEmpty": {
-    "open": true
+
```
Comment (KatarzynaQA) [inline cypress/fixtures/commonComponents/confirm.json:78]: Do we need this if we don't have nothing here ?

Comment (DlgSHi) [reply cypress/fixtures/commonComponents/confirm.json:78]: we have no `default` one object, so this is one for that purpose

File: .storybook/preview.js [hunk 3]
```diff
@@ -61,14 +56,17 @@ const customViewports = {
   },
 };
 
-addParameters({
+export const parameters = {
+  docs: {
+    theme: sageTheme,
+  },
+  layout: "fullscreen",
```
Comment (sjohnson-sage) [inline .storybook/preview.js:63]: Do we need `layout: "fullscreen"` here then`isFullScreen: false` in the options?

File: cypress/fixtures/commonComponents/confirm.json (hunk 2 above)
Comment (DlgSHi) [reply cypress/fixtures/commonComponents/confirm.json:78]: fixed

**Review by KatarzynaQA — DISMISSED** [review]

**Review by sjohnson-sage — DISMISSED** [review]

**Review by edleeks87 — DISMISSED** [review]

**Review by KatarzynaQA — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by DlgSHi — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by sjohnson-sage — APPROVED** [review]

**Pull request merged**